packages/vats/package.json

```diff
@@ -44,6 +38,8 @@
     "@agoric/vat-data": "^0.5.2",
     "@agoric/zoe": "^0.26.2",
     "@agoric/zone": "^0.2.2",
+    "@endo/captp": "^3.1.2",
+    "@endo/eventual-send": "^0.17.3",
```

Comment:
There's no occurrence of `@endo/captp` in the sources, and the single occurrence of `@endo/eventual-send` in `vats/test/bootstrapTests/supports.js` should be replaced with `@endo/far` (which is already in active use).

Reply:
> There's no occurrence of @endo/captp in the sources
removed
> the single occurrence of @endo/eventual-send in vats/test/bootstrapTests/supports.js should be replaced

That file moved to `@agoric/boot`. The dep is still in this package for:
https://github.com/Agoric/agoric-sdk/blob/50332a848618b9bec7fee9f565deea3a1e7827d3/packages/SwingSet/src/vats/plugin-manager.js#L4

Reply:
Why would a dependency of `packages/SwingSet` need to be surfaced in `packages/vats/package.json`?

Reply:
shoot I made [the mistake](https://github.com/Agoric/agoric-sdk/pull/8102#discussion_r1281128211) twice! https://github.com/Agoric/agoric-sdk/pull/8102#discussion_r1281128211

A stray Endo dep does no harm so I don't intend to run CI again to remove it. We have all of Endo in our `yarn.lock` so any package depending on it doesn't change runtime.